ui/imports/utils/Constants.qml

```diff
@@ -1438,8 +1438,9 @@ QtObject {
         readonly property string paraswapIcon: "paraswap"
         readonly property string paraswapHostname: "app.paraswap.io"
         readonly property string paraswapUrl: "https://www.paraswap.io/"
-        readonly property string paraswapApproveContractAddress: "0x216B4B4Ba9F3e719726886d34a177484278Bfcae"
-        readonly property string paraswapSwapContractAddress: "0xDEF171Fe48CF0115B1d80b88dc8eAB59176FEe57"
+        readonly property string paraswapV5ApproveContractAddress: "0x216B4B4Ba9F3e719726886d34a177484278Bfcae"
```

Comment:
@dlipicar the same question here?

Reply:
We want to keep compatibility with swaps generated with previous versions of the app, so until the dApp info comes in the Activity Entry, we probably want to keep the old address as well.